src/leetCodeExecutor.ts

```diff
@@ -74,8 +75,17 @@ class LeetCodeExecutor {
         );
     }
 
-    public async showProblem(id: string, language: string, outDir: string): Promise<string> {
-        return await this.executeCommandWithProgressEx("Fetching problem data...", "node", [await this.getLeetCodeBinaryPath(), "show", id, "-gx", "-l", language, "-o", `"${outDir}"`]);
+    public async showProblem(node: IProblem, language: string, outDir: string): Promise<string> {
+        const fileName: string = genFileName(node, language);
+        const filePath: string = path.join(outDir, fileName);
+        const hasLocalCode: boolean = await fse.pathExists(filePath);
```

Comment:
`hasLocalCode` is redundant. We can move `await fse.pathExists(filePath)` into the if block.

Reply:
OK.